### ISSUE TITLE
Disabling hot-corner plugin because core cannot suspend

### ISSUE THREAD

**Issue by pidlug** [gmc-holle/xfdashboard#197]
After upgrade to 0.9.4 from 0.9.3 (Arch packages from AUR), the hot-corner plugin stopped working. Here are messages displayed when I run xfdashboard in daemon mode:

```
$ xfdashboard -d

(xfdashboard:14022): xfdashboard-WARNING **: 09:00:24.558: Desktop ID 'cura.desktop' not found

(xfdashboard:14022): xfdashboard-WARNING **: 09:00:24.560: Desktop ID 'org.gnome.Music.desktop' not found

(xfdashboard:14022): xfdashboard-plugin-hot_corner-WARNING **: 09:00:24.564: Disabling hot-corner plugin because core cannot suspend.

(xfdashboard:14022): xfdashboard-WARNING **: 09:00:24.566: Desktop ID 'cura.desktop' not found

(xfdashboard:14022): xfdashboard-WARNING **: 09:00:24.568: Desktop ID 'org.gnome.Music.desktop' not found

(xfdashboard:14022): xfdashboard-WARNING **: 09:00:24.605: Malformed environment '--enable-crashpad' in environment set for PID 9821 at /proc/9821/environ
```
Downgrading to 0.9.3 fixed the issue, also "Disabling hot-corner plugin because core cannot suspend." is not displayed anymore.

**Comment by gmc-holle** [comment]
Hi,

thanks for your report. Yes, it found the bug which was introduced when removing the application class from libxfdashboard and put it to the real xfdashboard application. It was fixed and thanks to your report I found another minor bug ;)

Fixed with commit ddc8e81fcc12cabc1bee6308494518e43f53bb7f

**Comment by Guy-Dudeman** [comment]
I'm noticing that I'm getting that same "Disabling hot-corner plugin because core cannot suspend." message even though I'm using v.1.0.0... I see here that you say you've fixed it in v.1.1.0 but only v1.0.0 is in the .tar file. How can I upgrade to v1.1.0?
